frontend/src/ui/helpers/getLastNameHelper.ts

```diff
@@ -0,0 +1,33 @@
+const splitLastName = (lastName: string): string[] => lastName.split(' ');
+const isStartDe = (part: string): boolean => part.toLowerCase() === 'de';
+const isStartLa = (part: string): boolean => part.toLowerCase() === 'la';
+
+export const getMainLastNamePart = (lastNameParts: string[]): string => {
+  if (lastNameParts.length === 1) {
+    return lastNameParts[0];
+  }
+
+  if (isStartDe(lastNameParts[0])) {
+    if (isStartLa(lastNameParts[1]) && lastNameParts.length > 2) {
+      return `${lastNameParts[0]} ${lastNameParts[1]} ${lastNameParts[2]}`;
+    }
+    if (lastNameParts.length > 1) {
```

Comment:
I think this condition its unnecesary because here at this point, the lastNameParts always is higher than 1